shared/src/ledger/wallet/store.rs

```diff
@@ -257,24 +259,19 @@ impl Store {
         let (keypair_to_store, raw_keypair) = StoredKeypair::new(sk, password);
         let address = Address::Implicit(ImplicitAddress(pkh.clone()));
         let alias: Alias = alias.unwrap_or_else(|| pkh.clone().into()).into();
-        if self
```

Comment:
I think there was a bug here: in the case of alias collision, the user may modify the alias for the keypair, while the originally provided alias have been returned.

Reply:
ah yes, it was not possible to pick a different alias